keyboards/ghs/rar/readme.md

```diff
@@ -0,0 +1,15 @@
+# GHS.RAR
+
+![GHS.RAR](https://i.imgur.com/5n893ul.jpg)
+
+A seamless screwless 75% keyboard.
+
+* Keyboard Maintainer: [Ramon Imbao](https://github.com/ramonimbao)
+* Hardware Supported: ATmega32u4
+* Hardware Availability: [Gone Hacking Studio](https://gonehacking.studio)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make ghs/rar:default
```

Comment:
I'd add this to your other stuff.

`git mv keyboards/ghs keyboards/ramonimbao/`

```suggestion
    make ramonimbao/ghs/rar:default
```

Reply:
Ahh this particular keyboard is under the GHS group and not a personal project of mine.

Reply:
Okay